helix-core/src/shellwords.rs

```diff
@@ -3,8 +3,9 @@ use std::borrow::Cow;
 /// Get the vec of escaped / quoted / doublequoted filenames from the input str
 pub fn shellwords(input: &str) -> Vec<Cow<'_, str>> {
     enum State {
-        Normal,
-        NormalEscaped,
+        Delimiter, // whitespace
```

Comment:
To align with the other states I think we should call this "Delimited" or maybe "OnWhitespace"